.goreleaser.yaml

```diff
@@ -8,6 +8,9 @@ builds:
       - linux
       - windows
       - darwin
+    goarch:
```

Comment:
Hello @arthureichelberger! 
Thanks for opening your first Pull Request. 
We welcome you to the community!

Could you please add `386` to `goarch` list?

Reply:
The default value of goarch is already [ '386', 'amd64', 'arm64' ] ?

https://goreleaser.com/customization/builds/#builds

Reply:
Apologies for not noticing this earlier. I'm not sure what issue I had at the time, but if it's working for you, let's go ahead and close it. Thank you for the heads up @joebowbeer!